homeassistant/components/shopping_list/__init__.py

```diff
@@ -88,6 +97,18 @@ async def add_item_service(call: ServiceCall) -> None:
         if (name := call.data.get(ATTR_NAME)) is not None:
             await data.async_add(name)
 
+    async def remove_item_service(call: ServiceCall) -> None:
+        """Remove an item with `name`."""
+        data = hass.data[DOMAIN]
+        if (name := call.data.get(ATTR_NAME)) is None:
```

Comment:
`name` is required by the services schema, hence this should be:
```suggestion
        if (name := call.data[ATTR_NAME]) is None:
```
Same comment for the existing services.